Marlin/example_configurations/delta/FLSUN/auto_calibrate/Configuration.h

```diff
@@ -461,7 +461,7 @@
   #define DELTA_CALIBRATION_MENU
 
   // set the radius for the calibration probe points - max DELTA_PRINTABLE_RADIUS*0.869 if DELTA_AUTO_CALIBRATION enabled
-  #define DELTA_CALIBRATION_RADIUS ((DELTA_PRINTABLE_RADIUS) * 0.869) // mm
+  #define DELTA_CALIBRATION_RADIUS ((DELTA_PRINTABLE_RADIUS) * 0.869 - sqrt(sq(X_PROBE_OFFSET_FROM_EXTRUDER) + sq(Y_PROBE_OFFSET_FROM_EXTRUDER))) // mm
```

Comment:
Should use:
```cpp
HYPOT(X_PROBE_OFFSET_FROM_EXTRUDER, Y_PROBE_OFFSET_FROM_EXTRUDER)
```

Reply:
Hi LVD-AC

I tried your change however my probe (which is to the front and left of the nozzle) still probes off the bed. My code is as follows:

// uncomment to add G33 Delta Auto-Calibration (Enable EEPROM_SETTINGS to store results)
  #define DELTA_AUTO_CALIBRATION

  // NOTE NB all values for DELTA_* values MUST be floating point, so always have a decimal point in them

  #if ENABLED(DELTA_AUTO_CALIBRATION)
    // set the default number of probe points : n*n (1 -> 7)
    #define DELTA_CALIBRATION_DEFAULT_POINTS 4
  #endif

  #if ENABLED(DELTA_AUTO_CALIBRATION) || ENABLED(DELTA_CALIBRATION_MENU)
    // Set the radius for the calibration probe points - max DELTA_PRINTABLE_RADIUS*0.869 for non-eccentric probes
    #define DELTA_CALIBRATION_RADIUS ((DELTA_PRINTABLE_RADIUS) * 0.869 - sqrt(sq(X_PROBE_OFFSET_FROM_EXTRUDER) + sq(Y_PROBE_OFFSET_FROM_EXTRUDER)))
  #endif
................
#define X_PROBE_OFFSET_FROM_EXTRUDER -25     // X offset: -left  +right  [of the nozzle]
#define Y_PROBE_OFFSET_FROM_EXTRUDER -30   // Y offset: -front +behind [the nozzle]
#define Z_PROBE_OFFSET_FROM_EXTRUDER 1.4  // Z offset: -below +above  [the nozzle]

Wondering what I am doing wrong?

Please help.

Cheers,
George

Reply:
Also tried 

#define DELTA_CALIBRATION_RADIUS ((DELTA_PRINTABLE_RADIUS) * 0.869 - HYPOT(X_PROBE_OFFSET_FROM_EXTRUDER,Y_PROBE_OFFSET_FROM_EXTRUDER))

Exactly the same.

Reply:
Try setting the delta_calibartion_radius with M665 Bxx.xx till it stays on the bed

Reply:
that worked :)

Thank you very much.

READ: Iteration : 10                                    std dev:0.030
READ: .Height:278.11    Ex:-1.48  Ey:+0.00  Ez:-1.85    Radius:109.87
READ: .Tower angle :    Tx:-1.77  Ty:-1.37  Tz:+0.00

pretty happy with the std dev.

Reply:
did a calibration print and result was pretty poor.

back to the drawing board. :(

Reply:
Check your build, your tower angle corrections are rather large